files/initramfs-tools/union-mount.j2

```diff
@@ -11,6 +11,31 @@ case $1 in
     ;;
 esac
 
+set_tmpfs_log_partition_size()
+{
+  varlogsize=128
+
+  # NOTE: certain platforms, when reaching initramfs stage, have a small
+  #       limit of mounting tmpfs partition, potentially due to amount
+  #       of RAM available in this stage. e.g. Arista 7050 QX32
+  [ X"$aboot_platform" = X"x86_64-arista_7050_qx32"  ] && return
```

Comment:
>x86_64-arista_7050_qx32" [](start = 27, length = 24)

the best way to do this is to move the configuration part in device/platform/ , but this may requires more work.